modules/ROOT/pages/database-administration/syntax.adoc

```diff
@@ -62,6 +63,7 @@ GRANT READ
 
 Note that this command includes `+{+` and `+}+` in the syntax, and between them there can be a grouping of properties or the character `+*+`.
 It also has multiple optional parts, including the entity part of the command which is the grouping following the graph name.
+For details about the Graph privilege commands syntax, see xref:/authentication-authorization/manage-privileges.adoc#components-of-the-graph-privilege-commands[Components of the graph privilege commands].
```

Comment:
this bit isn't really about the graph type privilege itself, just using it to show usages of all special characters and to highlight the usage of the same characters being part of both syntax and the semantics of the syntax descriptions. Not sure we need to link it here, but I guess it doesn't hurt 🤷

Reply:
I know, but I also thought that it wouldn't hurt.